<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="camera" value="cam1"/>
    <arg name="serial_no" value="831612070975"/>
    <arg name="enable_depth" value="false"/>
    <arg name="enable_confidence" value="false"/>
  </include>

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="camera" value="cam2"/>
    <arg name="serial_no" value="831612072516"/>
    <arg name="enable_depth" value="false"/>
    <arg name="enable_confidence" value="false"/>
  </include>
</launch>

<!-- 831612072516 cam 2 -->